src/main/scala/ai/privado/languageEngine/javascript/passes/methodfullname/MethodFullNameForEmptyNodes.scala

```diff
@@ -31,10 +31,15 @@ import io.shiftleft.semanticcpg.language._
 
 class MethodFullNameForEmptyNodes(cpg: Cpg) extends ForkJoinParallelCpgPass[Call](cpg) {
   override def generateParts(): Array[Call] = {
-    cpg.call.methodFullName("").toArray
+    cpg.call.methodFullName("").toArray ++ cpg.call.methodFullName("<unknownFullName>").toArray
   }
 
   override def runOnPart(builder: DiffGraphBuilder, callNode: Call): Unit = {
-    builder.setNodeProperty(callNode, PropertyNames.MethodFullName, callNode.name)
+    val prefix = if (callNode.methodFullName == "") {
```

Comment:
@rahul-privado you aren't using `prefix` anywhere

Reply:
Not sure how I missed this. Used it now